resources/assets/pages/ShowUser.js

```diff
@@ -84,9 +66,7 @@ const ShowUser = () => {
         <h2 className="heading -emphasized -padded">
           <span>Campaigns</span>
         </h2>
-        {map(signups, signup => {
-          return <SignupCard key={signup.id} signup={signup} />;
-        })}
+        <SignupGallery userId={id} />
```

Comment:
This may be irrelevant to this work, but I notice we use `user.id` [above](https://github.com/DoSomething/rogue/pull/977/files#diff-34c9ce668130a04c6c47524a5004c9bcR56) and `id` here -- these should be the same ID, right? Would it make sense to make these consistent?

Reply:
Ah yep, they totally should. I updated these to both read from `user.id` in 85fc28d51f34bc6fc5927d8facfe86c6e12c5df9.

Reply:
I also added a "not found" handler in 6ac1565 just to be safe!